BlueprintUICommonControls/Sources/AccessibilityFocus.swift

```diff
@@ -0,0 +1,109 @@
+import BlueprintUI
+import UIKit
+
+/// Enables VoiceOver focus to jump to the wrapped element via a trigger that can be manually fired.
+public struct AccessibilityFocus: Element {
+
+    /// The element that will have the focus.
+    public var wrapped: Element
+
+    /// A object that can be held on to by the caller to manually trigger a focus.
+    public var trigger: Trigger
+
+    /// Creates a new `AccessibilityFocus` wrapping the provided element.
+    public init(
+        wrapping wrapped: Element,
+        trigger: Trigger
+    ) {
+        self.wrapped = wrapped
+        self.trigger = trigger
+    }
+
+    // MARK: Element
+
+    public var content: ElementContent {
+        ElementContent(child: wrapped)
+    }
+
+    public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
+        View.describe { config in
+            config.apply { view in
+                view.apply(model: self)
+            }
+        }
+    }
+}
+
+// MARK: Backing view
+
+extension AccessibilityFocus {
+
+
+    private final class View: UIView {
+
+        private var trigger: Trigger?
+
+        func apply(model: AccessibilityFocus) {
+            trigger?.action = nil
```

Comment:
Thinking out loud, should we do something like 

```swift
trigger?.action = {
  print("Trigger called after being reset. Set a breakpoint here to debug.")
}
```
Or something?

Reply:
I wouldn't bother, none of the other trigger implementations do this. It's more about cleaning up after ourselves than protecting against some programmer error.